frontend/src/components/TreeView.js

```diff
@@ -0,0 +1,179 @@
+import React, { useEffect, useState } from 'react'
+import { useNavigate, useLocation } from 'react-router-dom'
+import { Text, Box } from '@chakra-ui/react'
+import getAllChildren from '../utils/getAllChildren'
+
+const TreeView = (props) => {
+    const navigate = useNavigate()
+    const location = useLocation()
+    const { tickets } = props
+    const [tree, setTree] = useState({})
+    const pathname = location.pathname.substring(1).split('/')
+    const ticketType = pathname[0]
+    const ticketID = parseInt(pathname[1])
+
+    const TICKET_TYPES = Object.freeze({
+        SF: 'SF',
+        FI: 'FI',
+        PPR: 'PPR',
+        UPR: 'UPR',
+    })
+
+    const sortTickets = (list) => {
+        let sortedArr = list.sort((a, b) => (a.name > b.name ? 1 : -1))
```

Comment:
1. const sortedTickets = ...
2. Talked with @ansonjwhe, we decided best approach is to sort by id instead